SpywareFilter/sections/specific.txt

```diff
@@ -12,6 +12,8 @@
 !
 ||airunit.taobao.com/tracker.*.gif^
 ||cnn.com/optimizelyjs/*.js
+||ioam.de/iam.js
```

Comment:
In the past we blocked that file and then it got removed, don't know exact reason anymore. @Alex-302 can you enlighten?

Reply:
it causes a lot of issues